bigtable_rs/src/bigtable/read_rows.rs

```diff
@@ -33,6 +35,35 @@ pub async fn decode_read_rows_response(
     Ok(rows)
 }
 
+/// Decode the stream of `ReadRowsResponse` into a stream of `Result<(RowKey, Vec<RowCell>)>>`.
+pub async fn decode_read_rows_response_stream<'a>(
+    timeout: &'a Option<Duration>,
+    mut rrr: Streaming<ReadRowsResponse>,
+) -> Pin<Box<dyn Stream<Item = Result<(RowKey, Vec<RowCell>)>> + Send + 'a>> {
```

Comment:
I don't work with Pin often but my understanding is that since we're manipulating the stream, pinning is required here. I will play with it a little more to see if I can come up with something a little cleaner 👍🏻

Reply:
`&'a Option<Duration>` was complicating things significantly. Once I removed this parameter, we're able to use 'static and have a much nicer implementation. 👍🏻